template/docs/topics/development.md

```diff
@@ -0,0 +1,83 @@
+# Development
```

Comment:
Why is this file created? It's not being exposed in the repo as a README and is not included in any other file.

Reply:
As we discussed last week I plan for this to be exposed on the doc site. It's also ideal for scoped education reference.

Reply:
Same reply from `componentInstall.md`